### ISSUE TITLE
Documentation: create your first nn missing imports and dependencies, hard to follow

### ISSUE THREAD

**Issue by grexe** [Kotlin/kotlindl#563]
took me some time to figure out the needed libs and imports for getting the example to run - the example should mention this clearly:
https://github.com/Kotlin/kotlindl/blob/master/docs/create_your_first_nn.md

Getting TensorFlow to run from Kotlin on MacOS X ARM64 is another story... seems the only way is to compile it yourself:(

**Comment by grexe** [comment]
ah sorry this is actually mentioned in the setup already:
https://github.com/Kotlin/kotlindl#how-to-configure-kotlindl-in-your-project

don't know why I missed that, maybe add a back-reference in the guide still?

**Comment by zaleslaw** [comment]
Great idea @grexe will improve during this release cycle

**Comment by alexsavkaev** [comment]
Complete rookie here 
I just spent couple of hours trying to figure out the tutorial. Following code works only with org.jetbrains.kotlinx:kotlin-deeplearning-api:0.4.0 dependency or below.  And in stable version (currently 0.5.2) Sequential model is absent. Or I'm missing out how to reach it. 
```   
 val model = Sequential.of(
    Input(28,28,1),
    Flatten(),
    Dense(300),
    Dense(100),
    Dense(10)
)
```
Anyway, I'm just following steps of  [quick_start_guide.md](https://github.com/Kotlin/kotlindl/blob/master/docs/quick_start_guide.md)
